alembic/versions/7060183b85a8_init_migration.py

```diff
@@ -0,0 +1,49 @@
+"""Init migration
+
+Revision ID: 7060183b85a8
+Revises: 
+Create Date: 2024-12-16 19:12:15.210620
+
+"""
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision: str = '7060183b85a8'
+down_revision: Union[str, None] = None
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('city',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('name', sa.String(length=255), nullable=False),
+    sa.Column('additional_info', sa.String(length=511), nullable=True),
+    sa.PrimaryKeyConstraint('id'),
+    sa.UniqueConstraint('name')
+    )
+    op.create_index(op.f('ix_city_id'), 'city', ['id'], unique=False)
+    op.create_table('temperature',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('date_time', sa.DateTime(), nullable=True),
+    sa.Column('temperature', sa.Integer(), nullable=False),
+    sa.Column('city_id', sa.Integer(), nullable=True),
+    sa.ForeignKeyConstraint(['city_id'], ['city.id'], ),
```

Comment:
Ensure that the foreign key constraint on `city_id` is correctly enforced to maintain referential integrity between the `temperature` and `city` tables.